meta-mender-variscite/recipes-graphics/drm/libdrm_%.bbappend

```diff
@@ -0,0 +1 @@
+IMX_LIBDRM_SRC = "git://github.com/nxp-imx/libdrm-imx.git;protocol=https;nobranch=1"
```

Comment:
see above, misses machine override

Reply:
Updated in the commit faae79a1245fad558b1af9a126eddc0e1d5652fe